cmd/crc/cmd/root.go

```diff
@@ -148,3 +152,11 @@ func newMachine() machine.Client {
 func newMachineWithConfig(config crcConfig.Storage) machine.Client {
 	return machine.NewClient(constants.DefaultName, isDebugLog(), network.ParseMode(config.Get(cmdConfig.NetworkMode).AsString()))
 }
+
+func checkIfRunningAsNormalUser() error {
+	if os.Geteuid() != 0 {
+		return nil
+	}
+	logging.Debug("Ran as root")
+	return fmt.Errorf("crc should be ran as a normal user")
+}
```

Comment:
Note: Till now we have skip option for the preflight checks even for root user, with this change, it will be not available anymore. I am not sure we really required to have skip functionality for linux and mac.

Reply:
This is one of the `skip-` options which might be used by some people, so yeah, not sure it's a good thing to remove it.

Reply:
so we should remove this check totally ?

Reply:
@guillaumerose we added this check to force user to use crc as normal user mode instead admin/root, let's discuss this if we really want to remove it.

Reply:
https://github.com/code-ready/crc/pull/745 some discussion in the past when we made it skip-able I think it was because of windows.

Reply:
If the main problem is creating `~/.crc` with the wrong permissions, workaround could be to check if running as root and if  $HOME is `/root` or `/var/root` (on macOS). Another one could be to have an additional preflight check making sure ~/.crc is readable/writable the current user and fixing it.

Reply:
This is removing the check from preflight so the `skip` config will be gone, maybe we can make it skippable by using some environment variables `CRC_SKIP_ROOT_CHECK=true crc` so that users have a way to skip it if they want..